sdk/cognitiveservices/InkRecognizer/api/Azure.AI.InkRecognizer.netstandard2.0.cs

```diff
@@ -0,0 +1,211 @@
+namespace Azure.AI.InkRecognizer
```

Comment:
I don't think this should be here.

Reply:
Do you want me to disable the check for this project? It uses Azure.Core and has Track2 naming,

Reply:
I don't remember shipping a GA for it -- if we have, then we should keep the check.

Reply:
We plan to actually delete this project from the repo. So do whatever is easiest for this.

Reply:
We haven't shipped some other assemblies I have this enabled on. It's nice to be able to see what changes are being done to public API as part of PR.